backend/api/handlers/semester/GetEOS_ReportByIdHandler.go

```diff
@@ -5,25 +5,26 @@ import (
 	"fmt"
 	"net/http"
 
-	"LiteracyLink.com/backend/api/model"
 	"LiteracyLink.com/backend/db"
 	"github.com/gin-gonic/gin"
 	"gorm.io/gorm"
 )
 
 func GetEOS_ReportByIdHandler(c *gin.Context) {
 
-	var request model.GetEOS_Report
-	err := c.BindJSON(&request)
-	if err != nil {
+	ChildID := c.Param("child_id")
+	Semester_tutoring_obj := c.Param("semest_tut_obj_id")
```

Comment:
I don't think that the front end is ever handed the semest_tut_obj_id we should probably ask for the parent_id and then we can get the semest_tut_obj_id in the dao from the parent_id